src/resources/views/crud/inc/column_anchors.blade.php

```diff
@@ -0,0 +1,25 @@
+@php
+
+    if($column['anchor'] !== false) {
+        $related_model_key = $related_model_key ?? null;
+        $column['anchor']['href'] = isset($column['anchor']['href']) ? (is_callable($column['anchor']['href']) ? $column['anchor']['href']($crud, $column, $entry, $related_model_key) : ($column['anchor']['href'] ?? '')) : '';
+        $column['anchor']['class'] = isset($column['anchor']['class']) ? (is_callable($column['anchor']['class']) ? $column['anchor']['class']($crud, $column, $entry, $related_model_key) : ($column['anchor']['class'] ?? '')) : '';
+        $column['anchor']['target'] = $column['anchor']['target'] ?? '';
+    }
+@endphp
+
+@if($column['anchor'] !== false)
```

Comment:
Personally, after implementing Solution 3, I see this as becoming less repetitive, something like:

```php
@includeWhen(isset($column['wrapper']), 'crud::columns.inc.wrapperStart')
     @if(isset($escaped) && $escaped === false)
         {!! $text !!}
     @else
         {{ $text }}
     @endif
@includeWhen(isset($column['wrapper']), 'crud::columns.inc.wrapperEnd')
```

Reply:
Indeed, too easy to understand now.